pramen-py/src/pramen_py/metastore/writer_base.py

```diff
@@ -34,11 +34,11 @@ class MetastoreWriterBase(metaclass=abc.ABCMeta):
     """
 
     spark: SparkSession = attrs.field()
-    config: List[MetastoreTable] = attrs.field()
+    metastore_configs_tables: List[MetastoreTable] = attrs.field()
```

Comment:
```suggestion
    tables: List[MetastoreTable] = attrs.field()
```
I thought we agreed on 'tables'.

`metastore.metastore_configs_tables` is too long and looks a bit silly. If something is an attribute of metastore already, it should not have 'metestore' in its name'.